src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -80,31 +82,314 @@ public IList<MenuFlyoutItemBase> Items
+		//	TransitionCollection spTransitionCollection;
+		//	MenuPopupThemeTransition spMenuPopupChildTransition;
+
+		//	*transition = null;
+
+		//	(ctl.make(&spMenuPopupChildTransition));
+		//	(ctl.make(&spTransitionCollection));
+
+		//	spMenuPopupChildTransition.ClosedRatio = closedRatioConstant;
+
+		//	FrameworkElement overlayElement;
+		//	overlayElement = popup.OverlayElement;
+		//	spMenuPopupChildTransition.SetOverlayElement(overlayElement);
+
+		//	spTransition = spMenuPopupChildTransition;
+		//	spTransitionCollection.Append(spTransition));
+
+		//	// UNO TODO Windowed Popups are not supported
+		//	//
+		//	// For windowed popups, the transition needs to target the grandchild of the popup.
+		//	// Otherwise, the transition LTE is going to live in the main window and get clipped.
+		//	//if ((CPopup*)(popup.GetHandle()).IsWindowed())
+		//	//{
+		//	//	UIElement popupChild;
+		//	//	popupChild = popup.Child;
+
+		//	//	if (popupChild)
+		//	//	{
+		//	//		int childrenCount;
+		//	//		(VisualTreeHelper.GetChildrenCountStatic((UIElement*)(popupChild), &childrenCount));
+
+		//	//		if (childrenCount == 1)
+		//	//		{
+		//	//			xaml.IDependencyObject popupGrandChildAsDO;
+		//	//			(VisualTreeHelper.GetChildStatic((UIElement*)(popupChild), 0, &popupGrandChildAsDO));
+
+		//	//			if (popupGrandChildAsDO)
+		//	//			{
+		//	//				UIElement popupGrandChildAsUE;
+		//	//				(popupGrandChildAsDO.As(&popupGrandChildAsUE));
+		//	//				(popupGrandChildAsUE.Transitions = spTransitionCollection);
+		//	//				(popupGrandChildAsUE.InvalidateMeasure());
+		//	//			}
+		//	//		}
+		//	//	}
+		//	//}
+		//	//else
+		//	{
+		//		popup.ChildTransitions = spTransitionCollection;
+		//	}
+		//}
+
+		void AutoAdjustPlacement(MajorPlacementMode pPlacement)
+		{
+			// UNO TODO
+			// Rect windowRect = default;
+			// (DXamlCore.GetCurrent().GetContentBoundsForElement(GetHandle(), &windowRect));
+		}
+
+		public override void ShowAt(FrameworkElement pPlacementTarget)
+		{
+			m_openWindowed = false;
+			base.ShowAtCore(pPlacementTarget);
+		}
+
+		void ShowAtImpl(UIElement pTargetElement, Point targetPoint)
+		{
+			var targetElement = pTargetElement;
+
+			var showOptions = new FlyoutShowOptions();
+			showOptions.Position = targetPoint;
+
+			try
+			{
+				m_openingWindowedInProgress = true;
+
+				ShowAt(targetElement, showOptions);
+			}
+			finally
+			{
+				m_openingWindowedInProgress = false;
+			}
+		}
+
+		void CacheInputDeviceTypeUsedToOpen(UIElement pTargetElement)
+		{
+			// UNO TODO
+			//CContentRoot* contentRoot = VisualTree.GetContentRootForElement(pTargetElement);
+			//InputDeviceTypeUsedToOpen = contentRoot.GetInputManager().GetLastInputDeviceType();
+		}
+
+		// Callback for ShowAt() from core layer
+		void ShowAtStatic(MenuFlyout pCoreMenuFlyout,
+			UIElement pCoreTarget,
+		   Point point)
+		{
+			Debug.Assert(pCoreMenuFlyout != null);
+			Debug.Assert(pCoreTarget != null);
+
+			DependencyObject menuFlyout = pCoreMenuFlyout;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'menuFlyout'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)